code/src/components/AddTodo.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from 'reducers/todos'
+
+export const AddTodo = () => {
+  const [todoName, setTodoName] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(todos.actions.addTodo(todoName))
+    setTodoName('')
```

Comment:
To make this even better, think about handling an empty input. Right now it's possible to save a blank todo.
Maybe add something like:
`if (!todoName) return`
to make sure to return from the function if there's no todo written